common/djangoapps/student/models.py

```diff
@@ -1933,9 +1933,10 @@ class ManualEnrollmentAudit(models.Model):
     time_stamp = models.DateTimeField(auto_now_add=True, null=True)
     state_transition = models.CharField(max_length=255, choices=TRANSITION_STATES)
     reason = models.TextField(null=True)
+    role = models.CharField(blank=True, null=True, max_length=30)
```

Comment:
Can we increase max_length of role fields to 64 to make it consistent with other role fields defined in AccessRole model?